### DISCUSSION TITLE
Layout Problem

### DISCUSSION THREAD

**Discussion by paingchan** [filamentphp/filament#12168] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

<img width="1440" alt="image" src="https://github.com/filamentphp/filament/assets/54994337/93bb35a7-f2ce-4528-be63-b95c72db584b">
<img width="1440" alt="image" src="https://github.com/filamentphp/filament/assets/54994337/9a93dd75-0b45-4f82-87b4-d1d35c08b9fe">
How i can change to default layout ?

**Comment by leandrocfe** [comment, marked as answer]
Via css?
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

**Reply by paingchan** [reply]
Thank u bro i use php artisan make:filament-theme admin overwrite theme.css and tailwind.config.js
